### PR TITLE
Last fixes

### PR DESCRIPTION
**Pull request by ronnyleibham**
last fixes & css styling

### CODE REVIEW

File: Lebenslauf-Website/index.html [hunk 1]
```diff
@@ -90,22 +90,41 @@ <h2 class="text-white bg-black text-3xl pl-5 pr-5 pt-1 pb-2 mt-20 mb-9">
...
-          <li>
-            <a href="#"><img src="images/xing.png" alt="" />Xing</a>
-          </li>
-          <li>
-            <a href="#"><img src="images/github.png" alt="" />Github</a>
-          </li>
-        </ul>
+        <div
+          class="
+            grid grid-cols-3
```
Comment (F-Kirchhoff) [inline Lebenslauf-Website/index.html:95]: why do you use grid and flex together?

**Review by F-Kirchhoff — COMMENTED** [review]
looks fine, maybe you can center the content of the grid cell with other methods?

**Comment by F-Kirchhoff** [comment]
very cool look of the website!

File: Lebenslauf-Website/index.html (hunk 1 above)
Comment (ronnyleibham) [reply Lebenslauf-Website/index.html:95]: the positioning here is for the following div containers AND their content. you know another way? i'd appriciate.

**Pull request merged**